### PR TITLE
changed the alignment of spam and readmore button in user profile

### LINKED ISSUE #5770: Profile Page Issues
Hey guys, two bugs to report on the Profile Page since the update.

From the Profile Page, I'm not able to use the tabs to get over to things that I've coauthored or to drafts. I encountered this problem first on Safari, switched to Chrome, same issue. Hovering over the button shows the url: https://publiclab.org/profile/mimiss#draft or https://publiclab.org/profile/mimiss#coauthored but going directly to those links doesn't bring up either item. Here's a photo of what the page looks like:

Screen Shot 2019-05-22 at 3 05 58 PM

Second issue is that the buttons on the page are wonky (specifically mark as spam or read more), both in Safari and Chrome. The text and button don't align well. The buttons do work.

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes #5770  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!
profile_page

### CODE REVIEW

**Comment by AnshitaVishwa** [comment]
@jywarren I have done the required changes.

**Comment by AnshitaVishwa** [comment]
@CleverFool77  I have already attached the screenshot of the change with this PR. Have a look! Thanks!

**Comment by CleverFool77** [comment]
Apologies @AnshitaVishwa. I seem to missed out this PR. Can you remove the conflicts please ?

File: app/views/notes/_notes.html.erb [hunk 1]
```diff
@@ -47,8 +47,8 @@
               </small></p>
             <% end %>
         <p style="display:inline; margin: auto">
-              <% if logged_in_as(['admin','moderator']) %><a style="width: 40%" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
-              <a style="width: 50%" class="btn btn-outline-secondary" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a>
+              <% if logged_in_as(['admin','moderator']) %><a style="width: 50%" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
```
Comment (CleverFool77) [inline app/views/notes/_notes.html.erb]:
```suggestion
              <% if logged_in_as(['admin','moderator']) %><a style="width: 50%; margin-bottom: 0.5em;" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
```
Maybe we can bit of margin below span button ? @AnshitaVishwa

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**